common/formatting/token_partition_tree.h

```diff
@@ -19,7 +19,9 @@
 #include <iosfwd>
 #include <vector>
 
+#include "common/formatting/basic_format_style.h"
 #include "common/formatting/format_token.h"
+#include "common/formatting/line_wrap_searcher.h"
```

Comment:
Is this header needed here, or only in the .cc file?

Reply:
Nope. Needed only in .cc. Fixed.